Sources/TurboConfig.swift

```diff
@@ -11,12 +11,14 @@ public class TurboConfig {
 
     /// Optionally customize the web views used by each Turbo Session.
     /// Ensure you return a new instance each time.
-    public var makeCustomWebView: WebViewBlock?
+    public var makeCustomWebView: WebViewBlock = { (configuration: WKWebViewConfiguration) in
+        return WKWebView(frame: .zero, configuration: configuration)
```

Comment:
We can drop the `return` statement here since there is only one line in the function.